prow/tide/tide.go

```diff
@@ -1145,6 +1150,22 @@ func (c *changedFilesAgent) prChanges(pr *PullRequest) config.ChangedFilesProvid
 	}
 }
 
+func (c *changedFilesAgent) batchChanges(prs []PullRequest) config.ChangedFilesProvider {
+	return func() ([]string, error) {
+		var result []string
+		for _, pr := range prs {
+			changes, err := c.prChanges(&pr)()
+			if err != nil {
+				return nil, err
+			}
+
+			result = append(result, changes...)
```

Comment:
nit: If multiple PRs touch the same file this function will return duplicate entries in the slice. I don't think duplicates will cause any unexpected behavior, but it would probably be cleaner to use a set here.